frontend/src/cljs/imas_seamap/map/events.cljs

```diff
@@ -25,61 +25,68 @@
         {:db db
          :put-hash (encode-state db)}))))
 
+(defn get-feature-info-request
+  [info-format request-id by-server size bounds x y point]
+  (let [layers->str   #(->> % (map layer-name) reverse (string/join ","))]
+   ;; http://docs.geoserver.org/stable/en/user/services/wms/reference.html#getfeatureinfo
+    (for [[server-url active-layers] by-server
+          :let [layers   (layers->str active-layers)
+                priority (->> active-layers
+                              (map :category)
+                              (map {:imagery     0
+                                    :habitat     1
+                                    :third-party 2})
+                              (apply min))]]
+      (let [params {:REQUEST       "GetFeatureInfo"
+                    :LAYERS        layers
+                    :QUERY_LAYERS  layers
+                    :WIDTH         (:x size)
+                    :HEIGHT        (:y size)
+                    :BBOX          (bounds->str bounds)
+                    :FEATURE_COUNT 5
+                    :STYLES        ""
+                    :X             x
+                    :Y             y
```

Comment:
Final request please -- would you mind merging develop into your branch?  I made some concurrent edits that will be clobbered by these lines otherwise (essentially, x/y/width/height is fixed, and bounds is calculated separately from the viewport)